# U.S. Medical Insurance Costs
> Author: Edgar Qualkenbush, II <br>
> Date: Oct. 4, 2022
***

## Goals of Project
1. Ask
1. Prepare
1. Process
1. Analyze
1. Act
***

### Ask
The purpose of this case study is to research and analyze insurance cost in the U.S. as provided and initially from codecademy <br>
Original .csv file is provided in this GitHub repository as well
<br>
For this study, I initially did not have any questions to ask, so I felt a need to look at what data was provided first to brain storm questions that could be answered with the data set

Below are the libraries I felt were necessary to import for this study
1. pandas - this was initally used to import the .csv file into a workable dataframe

In [1]:
import pandas as pd

In [7]:
insurance = pd.read_csv('insurance.csv')
print(insurance.head())
print(insurance.shape)

   age     sex     bmi  children smoker     region      charges
0   19  female  27.900         0    yes  southwest  16884.92400
1   18    male  33.770         1     no  southeast   1725.55230
2   28    male  33.000         3     no  southeast   4449.46200
3   33    male  22.705         0     no  northwest  21984.47061
4   32    male  28.880         0     no  northwest   3866.85520
(1338, 7)


***
From the dataframe, there are a couple questions we could ask:
1. How do any of the variable impact the charges?
1. Are there any correlations between any stat? If so, which are the strongest indicators?

Initially, I will only focus on region and smoker status impact on charges <br>
As the data starts to become clearer to me, I may change my focus

### Prepare
Preparation for the data was completed above as I needed to look at the data in a dataframe to figure out what could be gleaned from it <br>
What is known now:
1. The data was downloaded from a module in the Data Scientist: Analytics Specialist Pro Career Path from codecademy.com
1. The data is a simple 1338 x 7 table
1. Since the purpose of this data set is for practice purposes, the data is credible 